app/views/recommendations/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= bootstrap_form_tag url: '/recommendations' do |f| %>
+<%= f.select :cuisine, [["Italian", "italian"], ["Mexican", "mexican"], ["Indian", "indian"], ["American", "american"], ["Thai", "thai"], ["Greek", "greek"], ["Chinese", "chinese"], ["Japanese", "japanese"], ["Latin", "latin"], ["Vietnamese", "vietnamese"], ["BBQ", "bbq"], ["Korean", "korean"], ["French", "french"]],
```

Comment:
@linda-le1 @jordanholtkamp - To confirm, the first "Italian" is what the user sees in the drop down, and the second "italian" is what we are going to pass in via params to beet farmer? If so, we'll actually need to adjust the params based on the endpoint mapping. For example, we want the "Italian" selection to actually use a param of "ital" for the endpoint based on research. There is an endpoint mapping markdown file in beet farmer master for reference. I'm also happy to make these changes if that's easier -- just lmk if that's how it works!

Reply:
Here is the endpoint mapping doc: https://github.com/linda-le1/beet-farmer/blob/master/endpoint_mappings.md